### PR TITLE
add command to the Dockerfile

### PR DESCRIPTION
**Pull request by trowik**

### CODE REVIEW

**Review by winged — CHANGES_REQUESTED** [review]
`manage.py runserver` is good for development (that's why it's in `docker-compose.override.yml`), but in production (which is what the `Dockerfile` is for), you should use `uwsgi` instead. You can look at https://github.com/projectcaluma/caluma/ (`Dockerfile`, `uwsgi.ini`, `requirements.txt`) to see how to do it.

**Comment by trowik** [comment]
something like that?

File: Dockerfile [hunk 1]
```diff
@@ -19,4 +19,6 @@ RUN pip install -r $REQUIREMENTS --disable-pip-version-check
 
 ADD . /app
 
-EXPOSE 8000
\ No newline at end of file
+EXPOSE 8000
+
+CMD /bin/sh -c "./wait-for-it.sh -t $WAIT_FOR_IT_TIMER $DATABASE_HOST:$DATABASE_PORT -- app/manage.py migrate && app/manage.py && uwsgi"
```
Comment (winged) [inline Dockerfile:24]: There's a `manage.py` in there without parameter.. might wanna remove that :)

Comment (winged) [reply Dockerfile:24]: > something like that?

Much better :) Did you test it, did it build / start correctly?

Comment (trowik) [reply Dockerfile]: it did build and start correctly

Comment (trowik) [reply Dockerfile]: otherwise the checks would have failed

Comment (winged) [reply Dockerfile]: Very good :+1:

**Review by winged — APPROVED** [review]

**Pull request merged**